### DISCUSSION TITLE
Nothing happens when running New-BurntToastNotification

### DISCUSSION THREAD

**Discussion by sap64** [Windos/BurntToast#203] [Q&A]
Hi @Windos, 

I am running into an issue when trying to run BurntToast on my machine. When running the New-BurntToastNotification command, it seems to execute but nothing happens. There is no toast notification or no error. I am also running PS as admin. I am fairly new to PS scripting, 

Thank you

Note: My ExecutionPolicy for LocalMachine is set to Bypass.

**Comment by Windos** [comment]
This generally means Windows has been set not to show notifications.

First check to make sure you don't have focus assist turned on.

You'll also want to check the "Notifications & actions" settings in the Settings app.

![image](https://user-images.githubusercontent.com/6955786/172089654-436221ef-5b05-413f-999e-ccf1f63b1cca.png)

**Reply by Born2Slow** [reply]
I'm having the same issue as @sap64. Running New-BurntToastShoulderTap returns the same warnings as well as the expected notification. But when I run New-BurntToastNotification nothing happens. No notification and no error in PS.

![image](https://user-images.githubusercontent.com/122380610/211598132-55034d50-9a95-48b4-97ad-64586f2ae5cd.png)

Notifications are enabled in Settings>System>Notifications & Actions however, Windows Powershell does not appear on the "Get Notifications from these apps" list. 

![image](https://user-images.githubusercontent.com/122380610/211596223-16721c4a-ae06-49e7-806b-15574efdfe84.png)

I've ensured Windows Powershell has a start menu folder as @the-c0d3r suggested. 

![PSFolder](https://user-images.githubusercontent.com/122380610/211597216-0dce5655-8a07-40a0-9052-9a368c13c3d7.jpg)

I used O&OShutUp10 when I installed windows so I considered the idea that one of the settings it changed may have effected notifications. But after going through all of the settings it changes one by one none of the obvious settings are disabled.

Full disclosure I'm very new to using PS and I'm aware that the solution might be staring me in the face. I really appreciate any insight I can get!

EDIT: I'm running Windows 10 Home 22H2 with all latest updates.

**Comment by sap64** [comment]
Hi @Windos,
Thanks for getting back to me. I do have notifications enabled. I do get a notification when running the New-BurntToastShoulderTap
![2022-06-06_13-31-46](https://user-images.githubusercontent.com/8248497/172214329-bdef0947-7d86-4e55-aaf9-12fc8660faf9.png)
![2022-06-06_13-37-01](https://user-images.githubusercontent.com/8248497/172215079-e0485626-53e0-40f1-8eaf-38f050ee39d4.png)

**Comment by the-c0d3r** [comment]
If all the above solutions doesn't work, check your start menu folder, does it have the powershell shortcut or not. #152 
I was facing the same issue on server 2019, and saw that somehow my startmenu folder shortcut for powershell is gone, so I created it and put it there, and it suddenly works. 

```
$env:USERNAME\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
```

**Reply by CarlGao4** [reply]
But mine is not working...

**Comment by PeterKW** [comment]
Did this ever work?

**Comment by richie19rich77** [comment]
I spent ages with the same issue.  Had it working on a number of computers, but on some I would run new-burnttoastnotification and nothing, no error or toast notification.

Then I discovered that I had "Do Not Disturb" enabled.

<img width="185" alt="image" src="https://github.com/Windos/BurntToast/assets/7002899/beb59996-18e8-4494-a76e-2edc64f6b845">

Once disable the notification displayed fine.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
